test/shopify-cli/theme/dev_server_test.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+require "test_helper"
+require "shopify_cli/theme/dev_server"
+
+module ShopifyCLI
+  module Theme
+    class DevServerTest < Minitest::Test
+      def setup
+        super
+        @ctx = ShopifyCLI::Context.new
+        @theme = stub(
+          "Dev Server Testing",
+          root: ".",
+          id: 1234,
+          name: "DevServer Test",
+          shop: "test.myshopify.io",
+          editor_url: "https://test.myshopify.io/editor",
+          preview_url: "https://test.myshopify.io/preview",
+          live?: false,
+        )
+        ShopifyCLI::Theme::DevServer.ctx = @ctx
+      end
+
+      def test_abort_when_invalid_theme
```

Comment:
tests to make sure the current context throws an abort when passed an invalid `-t/--theme` parameter